corgi/tasks/rhel_compose.py

```diff
@@ -53,11 +60,16 @@ def get_builds(
         relations_query = relations_query.filter(product_ref=stream_name)
 
     processed_builds = 0
-    for build_id in relations_query.values_list("build_id", flat=True).distinct().iterator():
+    for build_id, b_type in (
+        relations_query.values_list("build_id", "build_type").distinct().iterator()
+    ):
         if not build_id:
             continue
-        if not SoftwareBuild.objects.filter(build_id=int(build_id)).exists():
+        if not SoftwareBuild.objects.filter(build_id=int(build_id), build_type=b_type).exists():
             logger.info("Processing Compose relation build with id: %s", build_id)
+            if build_type == SoftwareBuild.Type.CENTOS:
+                # We would have to update collector modules to avoid using build_id as primary key
+                raise ValueError("We don't support modular builds in composes for CENTOS")
```

Comment:
This shouldn't ever happen, should it? We only set build_type to SoftwareBuild.Type.BREW or SoftwareBuild.Type.KOJI above. Or did you mean b_type here?
```suggestion
            if b_type == SoftwareBuild.Type.CENTOS:
                # We would have to update collector modules to avoid using build_id as primary key
                raise ValueError("We don't support modular builds in composes for CENTOS")
```

Reply:
Yes, updated to use build_type as the name instead of build_type as the constant in this module is now moved to tasks.common and renamed BUILD_TYPE